src/test/java/ladder/domain/UserTest.java

```diff
@@ -0,0 +1,37 @@
+package ladder.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import ladder.DoNotCreateUser;
+import org.junit.jupiter.api.Test;
+
+public class UserTest {
+
+    @Test
+    void 참여자는_최소_2이상이어야_한다() {
+        assertThatThrownBy(() -> {
+            new Users("yeju");
+        }).isInstanceOf(IllegalArgumentException.class);
+    }
+
+
+    @Test
+    void 사람_이름은_쉼표를_기준으로_구분한다() {
+        assertAll(() -> {
+            assertThat(new Users("yeju,mimi,aiai"));
```

Comment:
assertThat 이후에 검증코드가 없어서 무엇을 검증하는지 파악하기 어려운 것 같아요
생성자 호출 시 예외가 발생하지 않는 것을 검증하면 어떨까요? 
```suggestion
            assertDoesNotThrow(() -> new Users("yeju,mimi,aiai"));
```